Makefile.features

```diff
@@ -22,20 +22,34 @@ include $(RIOTCPU)/$(CPU)/Makefile.features
 # Resolve FEATURES_ variables
 # Their value will only be complete after resolving dependencies
 
-# Features that are required by the application but not provided by the BSP and
-# features that are used but blacklisted (prepended with "!").
-# Having features missing may case the build to fail.
-FEATURES_MISSING = $(sort $(filter-out $(FEATURES_PROVIDED),$(FEATURES_REQUIRED)))
-
 # Features that are only optional and not required at the same time.
 # The policy is to by default use by features if they are provided by the BSP.
 FEATURES_OPTIONAL_ONLY = $(sort $(filter-out $(FEATURES_REQUIRED),$(FEATURES_OPTIONAL)))
 FEATURES_OPTIONAL_USED = $(sort $(filter $(FEATURES_PROVIDED),$(FEATURES_OPTIONAL_ONLY)))
 # Optional features that will not be used because they are not provided
 FEATURES_OPTIONAL_MISSING = $(sort $(filter-out $(FEATURES_PROVIDED),$(FEATURES_OPTIONAL_ONLY)))
 
+# Features that are used without taking "one out of" dependencies into account
+FEATURES_USED_SO_FAR = $(sort $(FEATURES_REQUIRED) $(FEATURES_OPTIONAL_USED))
+
+# Additionally required features due to the "one out of" dependencies
+FEATURES_REQUIRED_ADDITIONS = $(foreach ooo,$(FEATURES_REQUIRED_ANY),$(call _features_any,$(subst |, ,$(ooo)),$(ooo)))
```

Comment:
maybe something like `FEATURES_REQUIRED_ONE_OUT_OF`, I think in this case your first name proposal fits here really nice. But this is nitpicking.

Reply:
Done